index.html

```diff
@@ -8,6 +8,76 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <h1>Hello to noone</h1>
+  <div class="right-button">
+    <div class="signin-text">
+      <div>Already have an Intuit account?</div>
+      <a href="">I forgot my User ID or Password</a>
+    </div>
+    <button class="signin-btn">
+      <img src="/assets/images/icon-whitelock@3x.png" alt="lock-icon">
+      Sign In
+    </button>
+  </div>
+
+  <main class="main-wrapper">
+    <div class="main-logo-box">
+      <img src="/assets/images/intuit_logos_new.png" alt="main-logo" class="main-logo">
+    </div>
+
+    <div class="product-logos">
+      <img src="/assets/images/mint_logos_new.png" alt="mint">
+      <img src="/assets/images/qb_logos_new.png" alt="qb">
+      <img src="/assets/images/turbo_logos_new.png" alt="turbo">
+    </div>
+
+    <section class="signin-form">
+      <h1>Create an Intuit account</h1>
+      <h2>One account for everything Intuit, including Mint.</h2>
+      <a href="">Learn more</a>
+
+      <form action="">
+          <div class="form-group">
+            <label for="">Email address</label>
+            <input type="text">
+          </div>
+
+          <div class="form-group">
+            <label for="">Phone (recommended)</label>
+            <input type="text">
+            <small>Standard call, messaging or data rates may apply.</small>
+          </div>
+
+          <div class="form-group" >
+            <label for="password-control">Password</label>
+            <div id="password-control">
+              <input type="password" >
+              <span></span>
+            </div>
+          </div>
+
+          <button class="signup-btn"><img src="/assets/images/icon-whitelock@3x.png" alt="lock-icon"> Create Account</button>
```

Comment:
Remove the first forward slash from src attribute. Should be src="assets/images/ic...